packages/icons/icons/crop.svg

```diff
@@ -0,0 +1,11 @@
+<svg 
```

Comment:
need to make sure the icon gets added  to figma first, then via the same process added here

Reply:
adding it now to figma

Reply:
adding now here